RevenueCatUI/CustomerCenter/Views/CustomerCenterView.swift

```diff
@@ -43,29 +43,46 @@ public struct CustomerCenterView: View {
 
     private let mode: CustomerCenterPresentationMode
```

Comment:
Ideally, I would love to deprecate this and move it inside `NavigationOptions` somehow, but that's definitely out of scope